plugins/inputs/vsphere/README.MD

```diff
@@ -0,0 +1,141 @@
+# VMware vSphere Input Plugin
+
+The VMware vSphere plugin uses the vSphere API to gather metrics from multiple vCenter servers.
+ 
+* Clusters
+* Hosts
+* VMs
+* Data stores
+
+## Configuration
+
+```
+# Read metrics from one or many vCenters
+[[inputs.vsphere]]
+  ## List of vCenter URLs, including credentials.
+  ## <protocol>://<user>:<password>@<vcenter>/sdk
+  ## Note the "@" characted must be escaped as %40
+  ## e.g.
+  ##     https://administrator%40vsphere.local:password@vcenter.local/sdk
+  #
+  vcenters = [ "https://administrator%40vsphere.local:password@vcenter.local/sdk" ]
```

Comment:
Use dedicated username and password fields for the credentials.  What we have learned is that doing the correct escaping in the userinfo section is a pain point for users and that it is difficult to keep this information out of the logs.

It does mean that you can't have different passwords without redefining the plugin but this is usually not a major issue in practice.

Reply:
OK. This would change the architecture of the plugin a bit, but it's definitely doable. The idea was to allow a single instance to monitor multiple vCenters, but if the best practice is to have one plugin instance per monitored system, we can certainly change that.

Reply:
You could still have the plugin monitor multiple vcenters if they have the same credentials.  

Plugins can usually be simplified by supporting only a single remote host though, because you can remove most of the concurrency and let Telegraf handle it.  The tradeoff is potentially a larger configuration file.  I think the choice mostly hinges on how many remote services you expect someone to monitor.

Reply:
OK. That makes the change almost trivial. We'll get it done very soon!